### DISCUSSION TITLE
error when doing: hugo mod get razonyang/hugo-theme-bootstrap@master

### DISCUSSION THREAD

**Discussion by kurt-krueckeberg** [razonyang/hugo-theme-bootstrap#764] [General]
I am a newbie. I installed the bootstrap theme using the starter template and successfully followed all the steps except the first step of [Upgrade Theme](https://github.com/razonyang/hugo-theme-bootstrap-skeleton#upgrade-theme) 
`
 hugo mod get razonyang/hugo-theme-bootstrap@master`   

This gives these errors: 
```
go: razonyang/hugo-theme-bootstrap@master: malformed module path "razonyang/hugo-theme-bootstrap": missing dot in first path element
Error: failed to get ["-d" "razonyang/hugo-theme-bootstrap@master"]: failed to execute 'go [get -d razonyang/hugo-theme-bootstrap@master]': failed to execute binary "go" with args [get -d razonyang/hugo-theme-bootstrap@master]: go: razonyang/hugo-theme-bootstrap@master: malformed module path "razonyang/hugo-theme-bootstrap": missing dot in first path element
 *errors.errorString
```

I'm not sure if I even need to follow the Upgrade Themes steps?

**Comment by razonyang** [comment]
Hi, it's a typo, the command should be `hugo mod get github.com/razonyang/hugo-theme-bootstrap@master`.
I'll fix the docs later, sorry for the inconvenience.

**Reply by kurt-krueckeberg** [reply]
Thank you for the prompt reply.
